bciers/apps/reporting/src/app/components/attachments/AttachmentsForm.tsx

```diff
@@ -191,6 +240,53 @@ const AttachmentsForm: React.FC<Props> = ({
             request
           </li>
         </ul>
+        {isSupplementaryReport && (
+          <div className="mt-4 border-t pt-4">
+            <p>
+              Before clicking &apos;Save & Continue&apos;, please confirm that
+              you understand and agree with the following statements:
+            </p>
+            <div className="flex items-start mt-3">
+              <Checkbox
+                checked={confirmExistingAttachmentsRelevant}
+                onChange={(e) =>
+                  setConfirmExistingAttachmentsRelevant(e.target.checked)
+                }
+                inputProps={{
+                  "aria-labelledby":
+                    "confirm-existing-attachments-relevant-label",
+                }}
+              />
+              <label
+                id="confirm-existing-attachments-relevant-label"
+                className="ml-2"
+              >
+                I confirm that I have uploaded any attachments that are required
+                to be updated for the new submission of this report.
+              </label>
+            </div>
+            <div className="flex items-start mt-3">
+              <Checkbox
+                checked={confirmRequiredAttachmentsUploaded}
+                onChange={(e) =>
+                  setConfirmRequiredAttachmentsUploaded(e.target.checked)
+                }
+                inputProps={{
+                  "aria-labelledby":
+                    "confirm-required-attachments-uploaded-label",
+                }}
+              />
+              <label
+                id="confirm-required-attachments-uploaded-label"
+                className="ml-2"
+              >
+                I confirm that any previously uploaded attachments that have not
+                been updated are still relevant to the new submission of this
+                report.
+              </label>
+            </div>
```

Comment:
This really should be part of the form we submit, instead of extra html added at the end, but RJSF makes this a pain if there is other HTML in the middle. Maybe one day we could rig something with this https://react.dev/reference/react-dom/createPortal